docs/csharp/language-reference/keywords/for.md

```diff
@@ -32,16 +34,18 @@ By using a `for` loop, you can run a statement or a block of statements repeated
   
     -   If `i` is greater than 5, the condition evaluates to `false`, and you exit the loop.  
   
- Note that, if the initial value of `i` is greater than 5, the body of the loop doesn't run even once.  
+Note that, if the initial value of `i` is greater than 5, the body of the loop doesn't run even once.
+
+## Sections of a for statement
   
- Every `for` statement defines initializer, condition, and iterator sections. These sections usually determine how many times the loop iterates.  
+Every [for statement](/dotnet/csharp/language-reference/language-specification/statements#the-for-statement) defines *initializer*, *condition*, and *iterator* sections. These sections usually determine how many times the loop iterates.  
   
 ```csharp  
 for (initializer; condition; iterator)  
     body  
```

Comment:
not sure if it's this line that is causing the rest of the formatting here on GH to look funny but everything else seems to be interpreted as code

Reply:
It looks correct to me (in GH and on the review site)

Reply:
@mairaw I don't think I see what you're seeing. [It looks okay](https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/for?branch=pr-en-us-5405) to me?

![image](https://user-images.githubusercontent.com/24882762/40088595-91e2ea2a-585c-11e8-8589-c66d9f61168e.png)

Reply:
It’s probably an issue with the GitHub diff functionality.

![1605d2af-1c72-4e74-be11-4eef3b5544f0](https://user-images.githubusercontent.com/12971179/40090310-ed360bb0-5865-11e8-9800-8813cb15696b.jpeg)